baseband/mark5b/base.py

```diff
@@ -64,6 +92,40 @@ def read_frame(self):
                                     ref_time=self.ref_time, nchan=self.nchan,
                                     bps=self.bps)
 
+    def get_frame_rate(self):
+        """Returns the number of frames per second.
```

Comment:
`"""...frames per second in a Mark 5B file."""`

Reply:
I think that should be obvious if one opened one!